### PR TITLE
docs: add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kenjis**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Contributing to CodeIgniter4
```
Comment (datamweb) [inline CONTRIBUTING.md:1]: Contributing to CodeIgniter Shield

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+CodeIgniter is a community driven project and accepts contributions of
+code and documentation from the community.
+
+If you'd like to contribute, please read the [Contributing to CodeIgniter](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/README.md)
+in the [main repository](https://github.com/codeigniter4/CodeIgniter4).
+
+If you are going to contribute to this repository, please report bugs or send PRs
```
Comment (datamweb) [inline CONTRIBUTING.md:9]: [report bugs](https://github.com/codeigniter4/shield/issues/new?assignees=&labels=bug&template=bug_report.yml&title=Bug%3A+) or [send PRs](https://github.com/codeigniter4/shield/compare)

**Review by datamweb — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Contributing to CodeIgniter4
+
+CodeIgniter is a community driven project and accepts contributions of
+code and documentation from the community.
+
+If you'd like to contribute, please read the [Contributing to CodeIgniter](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/README.md)
```
Comment (MGatner) [inline CONTRIBUTING.md:6]:
```suggestion
If you'd like to contribute, please read [Contributing to CodeIgniter](https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/README.md)
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (MGatner) [reply CONTRIBUTING.md:9]: Edited to turn into a suggestion (it was displaying as links in comment).

**Review by MGatner — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kenjis) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (kenjis) [reply CONTRIBUTING.md]: Done.

**Review by datamweb — APPROVED** [review]
Thanks!

**Review by MGatner — APPROVED** [review]
🤘

**Pull request merged**